gnosis/gp-dune-bridge
# Adding api(3rd step): Adding maintenance loop for updating inMemoryDatabase

Adds a maintenance loop that rereads the dune-downloads, in case there was a new json object fetched from dune.

Testplan:
1. Run the api with `cargo run`
2. Check some value in the api, e.g. `http://127.0.0.1:8080/api/v1/profile/0x5ab290183b21559d8a7e55fc11920b4caadda138`
3. Modify the user_data.json file for that address
4. wait 3 secs.
5. Check the served value of the api again.
